src/doc/gui_manual/BuildingOnMac/Execution.rst

```diff
@@ -27,5 +27,8 @@ Once masonry is running, it will produce log files in the ``_logs`` directory. T
 
    Mansonry Logs in HTML format
 
-
+Note that if you already have a web browser running on your system, that
+the web browser may not be able to connect to the local web server created
+to serve up the log file. In this case you should exit your existing web
+server and try again.
```

Comment:
'browser' instead of 'server'?

Reply:
I changed the last instance of server to browser.

Reply:
It is running a web server, so I think the original text may have applied.